.github/workflows/release.yml

```diff
@@ -1,125 +0,0 @@
-      needs-quarkus-platform-main-pr: ${{ steps.release-qosdk.outputs.need_main_pr }}
-
-    steps:
-      - uses: radcortez/project-metadata-action@main
-        name: Retrieve project metadata
-        id: metadata
-        with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
-          metadata-file-path: '.github/project.yml'
-
-      - uses: actions/checkout@v4
-
-      - name: Retrieve Quarkus version
-        id: quarkus-version
-        run: echo "quarkus_version=$(./mvnw help:evaluate -Dexpression=quarkus.version -q -DforceStdout)" >> $GITHUB_OUTPUT
-
-      - uses: actions/setup-java@v4
-        with:
-          distribution: 'temurin'
-          java-version: 17
-          cache: 'maven'
-          server-id: ossrh
-          server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
-          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
-          gpg-passphrase: MAVEN_GPG_PASSPHRASE
-
-      - name: Configure Git author
-        run: |
-          git config --local user.email "action@github.com"
-          git config --local user.name "GitHub Action"
-
-      - name: Maven release ${{steps.metadata.outputs.current-version}}
-        id: release-qosdk
-        run: |
-          quarkus_major=$(echo ${{steps.quarkus-version.outputs.quarkus_version}} | cut -d . -f 1,2)
-          echo "QOSDK-defined Quarkus major version: ${quarkus_major}"
-          
-          quarkus_platform_version=$(curl -s https://registry.quarkus.io/client/platforms | jq -r '.platforms[0]."current-stream-id"')
-          echo "Current platform-defined Quarkus major version: ${quarkus_platform_version}"
-          
-          quarkus_platform_branch=main
-          if [ "${quarkus_major}" = "${quarkus_platform_version}" ]; then
-            quarkus_platform_branch="${quarkus_platform_version}"
-          fi
-          echo "quarkus_platform_branch=${quarkus_platform_branch}" >> $GITHUB_OUTPUT
-          
-          if [ "${quarkus_platform_branch}" = "main" ]; then
-            echo "quarkus-platform update will target main"
-            echo "need_main_pr=true" >> $GITHUB_OUTPUT
-          else
-            echo "quarkus-platform update will target ${quarkus_platform_branch}"
-            echo "Check if quarkus-platform also needs to be updated on main"
-            if [ "${{ github.base_ref }}" = "main" ]; then
-              echo "PR targets main so assume quarkus-platform main also needs to be updated"
-              echo "need_main_pr=true" >> $GITHUB_OUTPUT
-            else
-              echo "No need for PR on quarkus-platform main"
-              echo "need_main_pr=false" >> $GITHUB_OUTPUT
-            fi
-          fi
-          
-          git checkout -b release
-          ./mvnw -B release:prepare -Prelease -Darguments="-DperformRelease -Dno-samples -DskipTests" -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
```

Comment:
I see you're passing a `-Dno-samples` here, which is not done in the reusable workflow. Can you define it in a Maven profile?

Reply:
Though that raises an interesting question that I hadn't paid attention to, indeed. How can I not release my samples with this new process?

Reply:
https://github.com/quarkiverse/quarkus-ironjacamar/blob/main/pom.xml#L70-L73

Reply:
Hmm, though that means I need another profile to deactivate based on the property…

Reply:
According to https://maven.apache.org/guides/introduction/introduction-to-profiles.html#multiple-conditions, you can specify more than one activation. Plus you can also specify `!true` to match "any value but true" in the `no-*` properties you already have: 
 
```xml
    <profile>
      <id>no-docs</id>
      <activation>
        <property>
          <name>no-docs</name>
          <value>!true</value>     
        </property>
        <property>
          <name>performRelease</name>
          <value>!true</value>
        </property>
      </activation>
      <modules>
        <module>docs</module>
      </modules>
    </profile>
```

Reply:
IDEA doesn't agree that you can specify multiple properties. Also, the semantics is supposed to be AND, which isn't what I would like (OR semantics).